include/yaramod/types/symbol.h

```diff
@@ -34,9 +34,18 @@ class Symbol
 		Dictionary,
 		Function,
 		Structure,
-		Reference
+		Reference,
+		Undefined
 	};
 
+	/// @name Constructors
+	/// @{
+	Symbol(Symbol::Type type, const std::string& name, ExpressionType dataType, const std::string& documentation = "")
+		: _type(type), _name(name), _documentation(documentation), _dataType(dataType)
+	{
+	}
+	/// @}
```

Comment:
I needed this constructor public, so I could construct a new `Symbol` instance in `parser_driver.cpp` on line 1635.